test/type_prop.cpp

```diff
@@ -8647,6 +8647,27 @@ TEST(type_prop, avg_pool_3d_deduce_strided_padded_small)
     EXPECT_EQ(avg_pool->get_padding_above(), (Shape{6, 4, 5}));
 }
 
+TEST(type_prop, avg_pool_ceil_mode)
```

Comment:
We'll want some execution tests ASAP as well. Currently BEs will be silently ignoring this flag.

Reply:
i was thinking of modifying the interpreter kernel but wasn't sure how to handle the extra rows/columns (ignore/include) Will need to look at the frameworks.